help/changelog

```diff
@@ -1,3 +1,22 @@
+=== 4.1.0 (2020-12-06)
+
+* Use tabs instead of floating windows (#688)
+* Add image comparison features to details dialog (#683)
+* Add exclusion filters for file names (#705)
+* Change reference row background color (#701)
+* Save/Load directories in Directories (#706)
+* Workaround for EXIF IFD type mismatch in parsing function (#630, #698)
+* Progress dialog stuck at "Verified X/X matches" (#693, #694)
+* Fix word wrap in ignore list dialog (#687)
+* Fix updating result window action upon creation (#685)
+* Colorize details table differences, allow rows to be moved around (#682)
+* Fix loading Result of 'Scan Type: Folders' shows only '---' in every table cell (#676, #677)
+* Add shortcut description to mark selected action (#656)
+* Fix row trimming being too aggressive. (#655, #654)
+* Add a preference option to disable bold font on reference row (#646)
+* Use relative icon path for themes to override more easily (#746)
+* Fix various building and packaging issues
+
```

Comment:
This is the list I have for changes since the last release, I grouped them by type for the github release, I don't think that should be done here:

## Changes Since Last Release
### Features
* Show the shortcut for "mark selected" in results dialog (#656, #641)
* Add option to enable/disable bold font (#646, #314) 
* Colorize details table differences, allow moving rows (#682)
* Use tabs instead of separate windows (#688)
* Save / Load directories as XML (#706)
* Add image comparison features to details dialog (#683)
* Change reference row background color, and allow user to adjust the color (#701)
* Add the ability to use regex based exclusion filters (#705)
### Bug Fixes
* Fix issue with details and results dialog row trimming (#655, #654) 
* Fix issues with Python 3.8 compatibility  (#665) 
* Fix loading Result of 'Scan Type: Folders' shows only '---' in every table cell (#677, #676)
* Fix flake8 issues (#672)
* Fix word wrap in ignore list dialog (#687)
* Progress dialog stuck at "Verified X/X matches" (#693, #694)
* Fix issue with result window action on creation (#685)
* Workaround for EXIF IFD type mismatch in parsing function  (#630, #698)
### Build
* Update CI builds and add windows CI (#572, #669)
* Update Packaging for Ubuntu (#593)
* Minor Build Updates (#627, #575, #628, #614)
* Update to use newer pytest and expand flake8 checking, cleanup various Deprecation Warnings
* Add warnings to packaging script when files are not built (#691)
* Use relative icon path for themes to override more easily  (#746)

Reply:
Should I remove the changelog then? Or copy what you have?

Reply:
Nevermind, I'm slow, I'll update accordingly, thanks.